includes/admin/admin-commands.php

```diff
@@ -56,7 +56,7 @@ function acf_commands_init() {
 			$custom_post_types[] = array(
 				'name'         => $post_type['post_type'],
 				'all_items'    => $labels->all_items,
-				'add_new_item' => $labels->add_new_item,
+				'add_new_item' => ucfirst( strtolower( $labels->add_new_item ) ),
```

Comment:
As far as I understand, RTL languages don't have upper and lower case, so this shouldn't affect them, nor would they require different treatment; is that correct, @mcsf?

Reply:
We shouldn't do this 🙈 Operations like this tend to break when applied across different languages, as they make implicit assumptions that don't always apply to all languages.

In this particular change, it breaks for German, where nouns are _always_ capitalized: "book" is "Buch", so "Neues buch hinzufügen" would be very wrong 😬 

Basically, localized strings should be left alone (which is also the reason why `$labels` has so many seemingly redundant strings) -- our best course of action is to continue to use `$labels->add_new_item`, and to handle the case of the strings in the individual translations.

Reply:
Thanks Bernie, I suspected this could not play well in other languages 😅 

In this case, the `add_new_label` in English capitalizes both words, as seen in the `Add new X` button, whereas [core commands for adding new pages](https://github.com/WordPress/gutenberg/blob/1d0c43d99a972643b6d97efa3a33762e31759eef/packages/core-commands/src/admin-navigation-commands.js#L77C6-L77C34) don't reuse strings and have their own post/page-specific strings.

My best guess here would be to add a completely new label to avoid breaking others, which aligns with the seemingly redundant strings you mentioned. Do you have any other recommendations?

Reply:
> My best guess here would be to add a completely new label to avoid breaking others, which aligns with the seemingly redundant strings you mentioned. Do you have any other recommendations?

Do you mean a new label which would be the same as `add_new_label` but with different casing expectations?

Reply:
I originally meant this, but that's before realizing core `add_new_label` values have been updated in 6.8 to remove the `New` part, so what we should do is change the default value for `add_new_value` to align with core when creating CPTs in SCF.